microk8s-resources/wrappers/microk8s-refresh-certs.wrapper

```diff
@@ -73,6 +80,43 @@ function revert {
   $SNAP/microk8s-start.wrapper
 }
 
+function refresh_ca {
+  echo "Backing up CA cert and key under $BACKUP"
```

Comment:
Start this function by checking if the user provided the CA files instead of taking a back up and then restoring the old CA from the backup. Something like:
```
if ! [ -e ${CAPATH}/ca.crt ]
then
  fail fast
fi
```

Reply:
Ok will do.